python-package/geobr/utils.py

```diff
@@ -143,18 +147,27 @@ def load_gpkg(url):
     except Exception as e:
 
         raise Exception(
-            'Some interal url is broken.'
+            'Some internal url is broken.'
             'Please report to https://github.com/ipeaGIT/geobr/issues'
         )
-    
-    with tempfile.NamedTemporaryFile(suffix='.gpkg') as fp:
+
+    # This below does not work in Windows -- see the Docs
+    # Whether the name can be used to open the file a second time, while the named temporary file is still open,
+    # varies across platforms (it can be so used on Unix; it cannot on Windows NT or later).
+    # https://docs.python.org/2/library/tempfile.html
+
+    # with tempfile.NamedTemporaryFile(suffix='.gpkg') as fp:
```

Comment:
It looks like a good solution. I don't think that we have to keep the comment though.

Reply:
I believe we should keep the comment. this is good  practice for legacy documentation